doc/api/crypto.md

```diff
@@ -1168,8 +1171,8 @@ recent OpenSSL releases, `openssl list-cipher-algorithms` will display the
 available cipher algorithms.
 
 The `key` is the raw key used by the `algorithm` and `iv` is an
-[initialization vector][]. Both arguments must be `'utf8'` encoded strings or
-[buffers][`Buffer`].
+[initialization vector][]. Both arguments must be `'utf8'` encoded strings,
+[buffers][`Buffer`] or `Uint8Array`s.
```

Comment:
@TimothyGu you mean the `buffers` that was already there should be uppercase? makes sense, updated